mscore/CMakeLists.txt

```diff
@@ -269,7 +270,7 @@ add_executable ( ${ExecutableName}
       fotomode.cpp drumtools.cpp
       selinstrument.cpp editstafftype.cpp texttools.cpp
       editpitch.cpp editstringdata.cpp editraster.cpp pianotools.cpp mediadialog.cpp
-      workspace.cpp workspacedialog.cpp chordview.cpp
+      workspacedialog.cpp chordview.cpp
```

Comment:
Why do you think any of these changes make any difference to the Travis-CI failures?
More likely you're missing some includes

   #include "workspace.h"  
   #include <unordered_map> 

probably in mtest.cpp

Reply:
I don't think there is any compile-time dependency at all - the problem comes at the link stage.  To me that suggests it's not an issue of what is included within the source, but rather, something about the build rule for mtest.  Like, perhaps some object file got added to the list in a way that make it get included in mtests as well as in mscore itself.  So to me, it does seem natural to think the problem is likely in CMakeLists.txt, and is something along these lines.  Unfortunately I'm really not intimately familiar with how this all works.